### DISCUSSION TITLE
2006/03/19/%E8%AE%B0%E5%BD%95%E6%A2%A6%E5%A2%83

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#78] [Announcements]
# 2006/03/19/%E8%AE%B0%E5%BD%95%E6%A2%A6%E5%A2%83

最近可能是因为换了个环境，有几个晚上睡觉做梦都是情景清晰，内容连贯的故事。把昨晚的故事记录下来吧，说不定哪天又有一部电影和我做的梦差不多。

https://blog.qizhen.xyz/2006/03/19/%E8%AE%B0%E5%BD%95%E6%A2%A6%E5%A2%83

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Qizhen
2006年09月1日 07:54 编辑
：）  你分析得一点都没错，所谓日有所思，夜有所梦。不过做梦比现实要浪漫些。

回复
samuel
2006年08月31日 23:26 编辑
我觉得,你的这个梦,说明你的压力太大了.差的人,就会被社会所淘汰.所以,一定要,努力!!!!!!!

回复
ERICA
2006年05月7日 23:19 编辑
从来就没看懂过的BLOG突然出现通俗易懂类文章，真的很搞笑,希望你可以再梦个续集.

回复
Anny
2006年03月28日 23:24 编辑
^_^
 
 
好久没有看到这么有意思的梦了

回复
Ling
2006年03月27日 17:10 编辑
果然牛银，连做梦都嘎有创意。。。。。。
@.@

回复
xian
2006年03月22日 11:40 编辑
在网上查苯海拉明的化学结构。。。一不小心搜到这里来了。
看来这里是个多信息的地方。。。赞~~

回复
奕瑋
2006年03月22日 11:20 编辑
可以兼职写剧本了。：）

回复
Jing
2006年03月20日 09:01 编辑
应该请斯皮尔伯格执导，一定是部好莱坞大剧！
